### PR TITLE
issue#19 fix

### PR DESCRIPTION
**Pull request by ashg1910**

### CODE REVIEW

**Comment by ashg1910** [comment]
corresponding change in css-
https://github.com/jquery/jquery-wp-content/pull/354

File: pages/index.html [hunk 1]
```diff
@@ -62,20 +62,28 @@ <h2 class="block">Corporate Members</h2>
...
+					<div class="qunitjs small logo"></div>
+				</div>
+			</a>
+			<a href="//sizzlejs.com">
+				<div class="project-tile six columns color sizzle-red">
+					<div class="sizzlejs small logo"></div>
+				</div>
+			</a>
 		</section>
 	</section>
```
Comment (rxaviers) [inline pages/index.html]: I think you can't have a DIV inside of an A element.
- http://www.w3.org/TR/html401/struct/links.html (12.2) `%inline`
- http://www.w3.org/TR/html401/struct/global.html#edef-DIV `%flow = %block | %inline`

**Comment by ashg1910** [comment]
http://stackoverflow.com/questions/1827965/is-putting-a-div-inside-an-anchor-ever-correct

File: pages/index.html (hunk 1 above)
Comment (mzgol) [inline pages/index.html]: @rxaviers Don't link to HTML 4 specs. :) They may be seriously outdated, a lot of things changed in HTML5 (including this one AFAIK).

**Pull request closed without merging**